### PR TITLE
Mark TEP 0070 as implemented.

### PR DESCRIPTION
**Pull request by bendory**

### CODE REVIEW

File: teps/0070-tekton-catalog-task-platform-support.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-status: proposed
+status: implemented
 title: Platform support in Tekton catalog
 creation-date: '2021-06-02'
 last-updated: '2021-06-02'
```
Comment (jerop) [inline teps/0070-tekton-catalog-task-platform-support.md:5]: please update this date too

**Review by jerop — APPROVED** [review]

**Comment by jerop** [comment]
implementation: https://github.com/tektoncd/catalog/pull/772 and https://github.com/tektoncd/hub/pull/282

**Comment by Yongxuanzhang** [comment]
https://github.com/tektoncd/community/tree/main/teps#teps
Tep70 in readme should also be updated?

File: teps/0070-tekton-catalog-task-platform-support.md (hunk 1 above)
Comment (bendory) [reply teps/0070-tekton-catalog-task-platform-support.md]: Done.

**Comment by bendory** [comment]
> https://github.com/tektoncd/community/tree/main/teps#teps Tep70 in readme should also be updated? The lint check will fail if both of them are not updated

Thanks, done.

**Comment by bendory** [comment]
> implementation: [tektoncd/catalog#772](https://github.com/tektoncd/catalog/pull/772) and [tektoncd/hub#282](https://github.com/tektoncd/hub/pull/282)
> 
> @bendory please add these PRs to the references

Thanks, done.

**Review by dibyom — APPROVED** [review]

**Pull request merged**